src/store/getters.ts

```diff
@@ -45,14 +45,12 @@ export default {
   /** Is True if there is a blocker including firm compliance. */
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   hasBlocker (state: StateIF, getters: any): boolean {
-    // check for firm with compliance warning
-    if (getters.isFirm && getters.hasComplianceWarning) {
-      return true
-    }
-    // check for firm with missing info warning
-    if (getters.isFirm && getters.hasMissingInfoWarning) {
-      return true
-    }
+    // check for compliance warning
+    if (getters.hasComplianceWarning) return true
+
+    // check for missing info warning
+    if (getters.hasMissingInfoWarning) return true
```

Comment:
I moved the "isFirm" check into the warnings getters below. (I also removed them from the Dashboard getters.) This is to centralize the isFirm checks (which are also implemented in the BE). This should make is easier to apply these getters to non-firm businesses in the future.